### PR TITLE
Language definitions

### PR DESCRIPTION
**Pull request by huseyin-kilic**
Language definition services have been created.

### CODE REVIEW

File: migrations/mysql/sqls/20160218065033-generate-language-table-down.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+drop table DEF_LANGUAGE;
```
Comment (oguzhanyalcin) [inline migrations/mysql/sqls/20160218065033-generate-language-table-down.sql]: Please use if exists in the drop query

**Comment by oguzhanyalcin** [comment]
Postgresql migration files are missing please commit them

**Comment by huseyin-kilic** [comment]
Postgresql migration files have been added

File: common/models/language.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "plural": "languages",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true,
+    "mysql": {
+      "table": "DEF_LANGUAGE"
+    },
+    "postgresql": {
+      "schema": "DEFINITIONS",
```
Comment (oguzhanyalcin) [inline common/models/language.json]: lower cased schema and table names for postgresql.....

File: common/models/language.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "language",
```
Comment (oguzhanyalcin) [inline common/models/language.json]: model names must be upper cased

**Pull request merged**